libs/ngx-charts-on-fhir/src/lib/fhir-mappers/observation/simple-observation-mapper.service.ts

```diff
@@ -71,7 +71,7 @@ export class SimpleObservationMapper implements Mapper<SimpleObservation> {
       ],
       scale: merge({}, this.linearScaleOptions, {
         id: layerName,
-        title: { text: [layerName, resource.valueQuantity.code] },
+        title: { text: [layerName, resource.valueQuantity?.code ?? ''] },
```

Comment:
Should we remove the second array element when unit/code is missing, instead of adding an empty string?